### PR TITLE
Update validate-auth to chaos-mesh-validation-auth

### PR DESCRIPTION
**Pull request by anuragpaliwal80**
### What problem does this PR solve?
As `validate-auth` is quite common. Let's use `chaos-mesh-validate-auth` to avoid confusion and conflicts.

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

CHANGELOG

- [ ] I have updated the `CHANGELOG.md`
- [x] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by anuragpaliwal80** [comment]
@STRRL Please review.

File: helm/chaos-mesh/templates/validating-admission-webhooks.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ webhooks:
 apiVersion: {{ $webhookApiVersion }}
 kind: ValidatingWebhookConfiguration
 metadata:
-  name: validate-auth
+  name: chaos-mesh-validate-auth
```
Comment (cwen0) [inline helm/chaos-mesh/templates/validating-admission-webhooks.yaml:88]: `{{ template "chaos-mesh.validation" . }}`

Comment (anuragpaliwal80) [reply helm/chaos-mesh/templates/validating-admission-webhooks.yaml]: done

**Comment by cwen0** [comment]
> @cwen0 Should we update changelog.md as well for this case. If so, please suggest the format. Should I reflect thin under [Unreleased] -> Changed section ?

Yes, you can update it under [Unreleased] -> Changed section.

**Review by STRRL — APPROVED** [review]
LGTM!

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**